components/application-broker/internal/broker/provision.go

```diff
@@ -394,6 +425,56 @@ func (svc *ProvisionService) channelForApp(applicationName internal.ApplicationN
 	return svc.knClient.GetChannelByLabels(integrationNamespace, labels)
 }
 
+// Create a new policy
+func (svc *ProvisionService) createIstioPolicy(ns internal.Namespace) error {
+	policyName := fmt.Sprintf("%s%s", ns, policyNameSuffix)
+
+	policyLabels := make(map[string]string)
+	policyLabels["eventing.knative.dev/broker"] = "default"
+
+	svc.log.Infof("Creating Policy %s in namespace: %s", policyName, string(ns))
+
+	brokerTargetSelector := &istioauthenticationalpha1.TargetSelector{
+		Name: brokerTargetSelectorName,
+	}
+	filterTargetSelector := &istioauthenticationalpha1.TargetSelector{
+		Name: filterTargetSelectorName,
+	}
+	mtls := &istioauthenticationalpha1.MutualTls{
+		Mode: istioMtlsPermissiveMode,
+	}
+	peerAuthenticationMethod := &istioauthenticationalpha1.PeerAuthenticationMethod{
+		Params: &istioauthenticationalpha1.PeerAuthenticationMethod_Mtls{
+			Mtls: mtls,
+		},
+	}
+
+	policy := &istiov1alpha1.Policy{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      policyName,
+			Namespace: string(ns),
+			Labels:    policyLabels,
+		},
+		Spec: istioauthenticationalpha1.Policy{
+			Targets: []*istioauthenticationalpha1.TargetSelector{brokerTargetSelector, filterTargetSelector},
+			Peers:   []*istioauthenticationalpha1.PeerAuthenticationMethod{peerAuthenticationMethod},
+		},
+	}
+
+	_, createError := svc.istioClient.AuthenticationV1alpha1().Policies(string(ns)).Create(policy)
```

Comment:
you can change `createError` and `updateError` to `err` in this block of code. It will work just fine as scoping will do the job